check-content.js

```diff
@@ -20,6 +20,7 @@ const filesToExcludeFromSidebarCheck = [
   "runtime_suite_libraries/*",
   "runtime_suite_tools/*",
   "business_suite/*",
+  "self_hosted/*",
```

Comment:
isn't it better to have `self_hosted/installation-chart`?

Reply:
Yes, more specific is better for this case. Now I changed it into `self_hosted/installation-chart/*`